staging/src/k8s.io/code-generator/cmd/validation-gen/output_tests/multiple_tags/zz_generated.validations.go

```diff
@@ -44,8 +44,6 @@ func RegisterValidations(scheme *runtime.Scheme) error {
 }
 
 func Validate_T1(in *T1, fldPath *field.Path) (errs field.ErrorList) {
-	// TypeMeta
```

Comment:
I thought it was sort of interesting to emit the comment for every field (even better if we could say "TypeMeta has no validations".  Is that something you think is worthwhile or no?

Reply:
Oops, I didn't mean to remove it.  I thought it was a nice touch.